sources/packages/backend/apps/api/src/app.institutions.module.ts

```diff
@@ -42,6 +42,7 @@ import {
   EducationProgramOfferingInstitutionsController,
   EducationProgramOfferingControllerService,
   ConfirmationOfEnrollmentControllerService,
+  StudentInstitutionController,
```

Comment:
StudentInstitution**_s_**Controller?

While using student or institution as client type we use plural names